HW03_Julia.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+title: "HW03_Julia"
+author: "Julia Shangguan"
+date: "7/19/2020"
+output: github_document 
+---
+
+```{r setup, include = TRUE}
+knitr::opts_chunk$set(echo = TRUE, error = TRUE)
+```
+
+## HWO3: Looking at COVID-related data
+
+There is a lot of cool data available from the Pew Research Center. Here is where I downloaded my dataset from: [PewResearchCenter](https://www.journalism.org/dataset/covid-19-late-april-2020/)
+
+**Loading in necessary packages and installing packages:**
+```{r load libraries}
+install.packages('survey') #install this package for analysis of data analysis
+library(survey)
+install.packages('tidyverse')
+library(tidyverse)
+library("ggplot2")
+library(foreign) #needed to import my dataset
+library(scales) #needed to later convert my x-axis labels from decimal to percent
+```
+
+**Importing my dataset as an R data.frame:**
+```{r}
+Covid_data <- read.spss("Pathways April 2020 (ATP W66).sav", to.data.frame = TRUE) 
```

Comment:
I think it would be better to have the dataset imported from a URL; I was unable to download the dataset without a login to the Pew Research Center.

Reply:
I think that uploading the dataset to github would work as well.